### ISSUE TITLE
Add an API architecture diagram and how to access the Swagger UI for v3.0.0

### ISSUE THREAD

**Issue by FeynmanZhou** [kubesphere/website#566]
Example: 

https://v2-1.docs.kubesphere.io/docs/api-reference/api-docs/
https://v2-1.docs.kubesphere.io/docs/api-reference/api-guide/#how-to-access-swagger-ui

- [ ] An API architecture diagram 
- [ ] How to access the Swagger UI

**Comment by RolandMa1986** [comment]
@FeynmanZhou  We don't provide Swagger UI anymore in 3.0.0. However, we provide an OpenAPI document https://kubesphere.io/api/kubesphere/.  We can add the link to https://kubesphere.io/docs/api-reference/api-docs/

**Comment by FeynmanZhou** [comment]
> @FeynmanZhou We don't provide Swagger UI anymore in 3.0.0. However, we provide an OpenAPI document https://kubesphere.io/api/kubesphere/. We can add the link to https://kubesphere.io/docs/api-reference/api-docs/

Yes, but the CRD APIs are not included in `https://kubesphere.io/api/kubesphere/`. Please provide an API architecture diagram for v3.0.0.
